LICENSE

```diff
@@ -1 +1,21 @@
-TODO: Add your license here.
+The MIT License
```

Comment:
I've always used the BSD 2-clause but i guess [they're pretty much the same](https://opensource.stackexchange.com/questions/217/what-are-the-essential-differences-between-bsd-and-mit-licences)